src/main/java/net/openhft/chronicle/bytes/MappedFile.java

```diff
@@ -46,6 +47,7 @@
 public abstract class MappedFile extends AbstractCloseableReferenceCounted {
     static final boolean RETAIN = Jvm.getBoolean("mappedFile.retain");
     private static final long DEFAULT_CAPACITY = 128L << 40;
+    // The canonical path is pre-pended with static and random data to prevent unrelated synchronization on internalized Strings
     protected final String canonicalPath;
```

Comment:
Maybe it should be called pathLock? It's not path anymore.

Reply:
Yepp. I thought that too but then the compatibility plugin complained as this variable is in the API. We can revisit once we have internalized it.

Reply:
In this case it is better to rename it, if any users of this class happen to use this variable as a path, they should definitely rewrite their code now, since it is no good anymore. It has protected access so renaming will no longer be purely decorative.

Compatibility plugin deliver value but has to be tuned down.

Reply:
After it is renamed to a descriptive name the comment may be shortened or dropped.

Reply:
This is fixed now